### ISSUE TITLE
Add ability to modify the ga('create'...)

### ISSUE THREAD

**Issue by maticrivo** [tcoopman/ga-react-router#10]
We should have the ability to send additional data to the create method.
https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#create

**Comment by tcoopman** [comment]
Good idea. If you are still interested in this, can you provide a pull request?

**Comment by maticrivo** [comment]
I'll see if i can work something out this weekend

**Comment by maticrivo** [comment]
I wrote something up, couldn't test it yet, but you can take a look at pull request #14
